lib/checkPullRequestLabels.js

```diff
@@ -43,29 +43,6 @@ var matchChangelogLabelWithRegex = function(changelogLabel) {
   return regExpForChangelogLabel.test(changelogLabel);
 };
 
-/**
- * This function checks if the project owner can be assigned to a PR.
- *
- * @param {import('probot').Octokit.PullsGetResponse} pullRequest
- * @param {String} changelogLabel
- */
-const canAssignProjectOwner = (pullRequest, changelogLabel) => {
```

Comment:
Why are we moving this to utils?

Reply:
I think this was done before I decided to assign the last reviewer. I'll move it back here.